enhancements/ingress/customize-error-code-pages.md

```diff
@@ -0,0 +1,147 @@
+---
+title: customize-error-code-pages
+authors:
+  - "@miheer"
+reviewers:
+  - "@Miciah"
+  - "@danehans"
+  - "@knobunc"
+approvers:
+  - "@Miciah"
+  - "@danehans"
+  - "@knobunc"
+creation-date: 2021-03-07 
+last-updated: 2020-03-07
+status: implementable 
+see-also:
+replaces:
+superseded-by:
+---
+# Customize error code paged returned by the haproxy router
+## Release Signoff Checklist
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+## Summary
+There is no supported method to customize an IngressController's error pages in OCP 4.
+Users may want to customize an IngressController's error pages for branding or other reasons.
+For example, users may want a custom HTTP 503 error page to be returned if no pod is available.
+When the requested URI does not exist, users may want an IngressController to return a custom 404 page.
+## Motivation
+The primary motivation is that users want to customize the error pages that are returned, for example for branding purposes.
+Secondary motivation is that users say 503 page is served even when the page is not found when it shall serve 404.
+The reason that happens is because we provide only 503 in our haproxy template.
+
+### Goals
+Return a distinct HTTP 404 error page when the requested URI does not exist, instead of returning the HTTP 503 error page.
```

Comment:
Looks like you accidentally deleted the old goal: "Enable the cluster administrator to specify custom error code pages for haproxy router".